ext/pdo_firebird/pdo_firebird.c

```diff
@@ -80,8 +80,12 @@ PHP_MSHUTDOWN_FUNCTION(pdo_firebird) /* {{{ */
 
 PHP_MINFO_FUNCTION(pdo_firebird) /* {{{ */
 {
+	char tmp[64], *s;
+	isc_get_client_version(s = tmp);
```

Comment:
The `s` variable seems to be superfluous; you can use `tmp` below. And maybe renaming it to `version` would clarify the intent.